src/Editor/GUI/Editors/ModelEditor.cpp

```diff
@@ -46,16 +46,29 @@ void ModelEditor::Show() {
 
         if (hasSourceFile) {
             ImGui::Text("Mesh Data");
+            ImGui::Checkbox("Uniform Scaling", &uniformScaling);
+
+            if (uniformScaling) {
+                float uniScale = scale.x;
+                ImGui::DragFloat("Scale", &uniScale, 0.01f);
+                scale = glm::vec3(uniScale);
+            }
+            else
```

Comment:
Please put the else keyword on the same line as the previous brace and wrap the else body in braces according to style guidelines.